compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/pointer/PointerEvent.skiko.kt

```diff
@@ -339,17 +343,71 @@ actual val PointerKeyboardModifiers.isScrollLockOn: Boolean
 actual val PointerKeyboardModifiers.isNumLockOn: Boolean
     get() = (packedValue and KeyboardModifierMasks.NumLockOn) != 0
 
-@OptIn(ExperimentalComposeUiApi::class)
-internal fun PointerButtons.copyFor(
+internal fun PointerKeyboardModifiers.update(
```

Comment:
I wouldn't extract this logic into separate `update` functions, it belongs only to `DefaultPointerStateTracker`, not as a broader helper function.

The initial purpose of this logic was to guess the current state by incoming events in case of small input data.

Reply:
The purpose is still the same. I'll move it to `DefaultPointerStateTracker`